website/content/middleware/decompress.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Decompress Middleware"
+description = "Decompress middleware for Echo"
+[menu.main]
+  name = "Decompress"
+  parent = "middleware"
++++
+
+Decompress middleware decompresses HTTP request if Content-Encoding header is set to gzip.
```

Comment:
Should we add the information here, that the decompressed body will consume memory which needs to be allocated?

Reply:
yap, LGTM